### PR TITLE
Update "Ethereum" to more relevant terminology

### PR DESCRIPTION
**Pull request by john-light**

### CODE REVIEW

File: docs/sdk/lib-ethers.ethersliquity.connect_1.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ static connect(signerOrProvider: EthersSigner | EthersProvider, optionalParams?:
 
 |  Parameter | Type | Description |
 |  --- | --- | --- |
-|  signerOrProvider | [EthersSigner](./lib-ethers.etherssigner.md) \| [EthersProvider](./lib-ethers.ethersprovider.md) | Ethers <code>Signer</code> or <code>Provider</code> to use for connecting to the Ethereum network. |
+|  signerOrProvider | [EthersSigner](./lib-ethers.etherssigner.md) \| [EthersProvider](./lib-ethers.ethersprovider.md) | Ethers <code>Signer</code> or <code>Provider</code> to use for connecting to the EVM-compatible network. |
```
Comment (tjcloa) [inline docs/sdk/lib-ethers.ethersliquity.connect_1.md:19]: it says "don't edit this file..." in the beginning

File: packages/lib-ethers/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Install in your project:
 npm install @sovryn-zero/lib-base @sovryn-zero/lib-ethers ethers@^5.0.0
 ```
 
-Connecting to an Ethereum node and sending a transaction:
+Connecting to a blockchain node and sending a transaction:
```
Comment (tjcloa) [inline packages/lib-ethers/README.md:13]: this one will do but it is out of my scope - i'm  only fixing contracts/contracts 
this one is typescript SDK folders which is managed by CoL

**Review by tjcloa — COMMENTED** [review]
i'm good with this 
but the docs file will be rewritten - it is generated automatically

File: docs/sdk/lib-ethers.ethersliquity.connect_1.md (hunk 1 above)
Comment (john-light) [reply docs/sdk/lib-ethers.ethersliquity.connect_1.md:19]: That's annoying. Is that because it will be overwritten?

Comment (tjcloa) [reply docs/sdk/lib-ethers.ethersliquity.connect_1.md:19]: yes
the docs are auto-generated

**Pull request closed without merging**